flood_forecast/custom/custom_opt.py

```diff
@@ -14,27 +16,91 @@ def warmup_cosine(x, warmup=0.002):
         return x/warmup
     return 0.5 * (1.0 + torch.cos(math.pi * x))
 
+
 def warmup_constant(x, warmup=0.002):
     """ Linearly increases learning rate over `warmup`*`t_total` (as provided to BertAdam) training steps.
         Learning rate is 1. afterwards. """
     if x < warmup:
         return x/warmup
     return 1.0
 
+
 def warmup_linear(x, warmup=0.002):
     """ Specifies a triangular learning rate schedule where peak is reached at `warmup`*`t_total`-th (as provided to BertAdam) training step.
         After `t_total`-th training step, learning rate is zero. """
     if x < warmup:
         return x/warmup
     return max((x-1.)/(warmup-1.), 0)
 
+
 SCHEDULES = {
     'warmup_cosine':   warmup_cosine,
     'warmup_constant': warmup_constant,
     'warmup_linear':   warmup_linear,
 }
 
 
+class RMSELoss(torch.nn.Module):
```

Comment:
I would suggest putting these loss functions in a separate file called `loss` (i.e. `flood_forecast/custom/loss.py`: `custom_opt` should be all about optimizers...)

Reply:
Yeah that is a good point. But I might address that in a future PR.